java/vector/src/test/java/org/apache/arrow/vector/TestVectorReset.java

```diff
@@ -136,6 +140,13 @@ public void testListTypeReset() {
       fixedList.setNull(0);
       fixedList.setValueCount(1);
       resetVectorAndVerify(fixedList, fixedList.getBuffers(false));
+
+      // ListViewVector
```

Comment:
@lidavidm I didn't add the `LargeList` and `LargeListView` as `LargeList` wasn't there in the first place.